lib/Frontend/ParseableInterfaceSupport.cpp

```diff
@@ -265,6 +265,9 @@ static bool buildSwiftModuleFromSwiftInterface(
     for (auto const &Dep : DepNames) {
       auto DepStatus = FS.status(Dep);
       if (!DepStatus) {
+        Diags.diagnose(SourceLoc(),
+                       diag::missing_dependency_of_parseable_module_interface,
```

Comment:
Ah, I think it's worth printing the error that occurred too; the dependency might not actually be missing.

Reply:
Fixed.